2차/류동준_시저-암호.py

```diff
@@ -0,0 +1,22 @@
+import string
+from collections import OrderedDict
+
+def solution(s, n):
+    alphabets = OrderedDict(zip(string.ascii_lowercase,range(1,27)))
+    result = []
+    for c in s:
+        if c.isalpha():
+            index = (alphabets[c.lower()] + n) % 26
+            if index == 0:
+                index = 26
+            for key,value in alphabets.items():
+                if value == index:
+                    tmp = key
+            if c.isupper():
+                tmp = tmp.upper()
+            elif c.islower():
+                tmp = tmp.lower()
+            result.append(tmp)
```

Comment:
```suggestion
            result.append(tmp.upper() if c.isupper() else tmp.lower())
```
어처피 문자열에서 lower와 upper 둘 중 하나이기 때문에 if-elif로 묶을 필요가 없을 듯 해요